src/lib/docs/Antelope/Bytes.md

```diff
@@ -3,24 +3,23 @@ title: Bytes
 description: The byte representation used internally for all Antelope data types and serialization
 category: Antelope
 published: true
-requiresReview: true
 ---
 
 # Bytes
 
-The `Bytes` type is the internal type used for the majority of other Antelope types and is regularly used in conjunction with the [Serializer](/docs/antelope/serializer). The `Bytes` type is capable of wrapping hex, utf-8, and native JavaScript buffers and arrays which it internally stores as an `Uint8Array`.
+The `Bytes` type is the internal type used for the majority of other Antelope types and is regularly used in conjunction with the [Serializer](/docs/antelope/serializer). The `Bytes` type is capable of wrapping hex, utf-8, and native JavaScript buffers and arrays, which it internally stores as a `Uint8Array`.
 
 ## Usage
 
-The `Bytes` type acts as the base data format for many other Antelope types, and so occasionally developers will need to coerce their data into one of the `BytesType`. This conversion can be performed by using the `Bytes.from` method and passing in any of the supported types].
+The `Bytes` type acts as the base data format for many other Antelope types, and so occasionally developers will need to coerce their data into one of the `BytesTypes`. This conversion can be performed by using the `Bytes.from` method and passing in any of the supported types.
```

Comment:
I'm also going to assume that `BytesType` here is predefined and can't be changed to `BytesTypes`. Maybe it could read, "... one of the `BytesType` values" instead?

Reply:
Correct!

Reply:
Fixed